kotlinpoet/src/main/java/com/squareup/kotlinpoet/KModifier.kt

```diff
@@ -54,6 +54,7 @@ public enum class KModifier(
 
   ENUM("enum", Target.CLASS),
   ANNOTATION("annotation", Target.CLASS),
+  VAL("val", Target.CLASS),
```

Comment:
Isn't it `value`? https://kotlinlang.org/docs/inline-classes.html

Reply:
D'oh, good catch